hazelcast/db.py

```diff
@@ -329,6 +419,21 @@ def connect(
     port: int = None,
     cluster_name: str = None,
 ) -> Connection:
+    """Creates a new Connection to the cluster
+
+    Args:
+         config: A Config object
+         dsn: Dota Source Name in the following formaT:
+         `hz://[user:password]@addr1:port1[?opt1=value1[&opt2=value2 ...]]`
+         user: Optional user name for authenticating to the cluster.
+         password: Optional password for authenticating to the cluster.
+         host: Hostname or IP address of the cluster. By default `localhost`.
+         port: Port of the cluster. By default `5701`.
+         cluster_name: Name of the cluster. By default `dev`.
```

Comment:
```suggestion
         host: Hostname or IP address of the cluster. By default ``localhost``.
         port: Port of the cluster. By default ``5701``.
         cluster_name: Name of the cluster. By default ``dev``.
```
to be consistent with other docs. same for other places